packages/controllers/src/claim.rs

```diff
@@ -582,4 +604,50 @@ mod test {
 
         assert_eq!(queried_claims.claims.len(), 0);
     }
+
+    #[test]
+    fn test_range_query() {
+        let mut deps = mock_dependencies();
+        let claims = Claims::new("claims");
+
+        claims
+            .create_claim(
+                deps.as_mut().storage,
+                &Addr::unchecked("addr"),
+                TEST_AMOUNT.into(),
+                TEST_EXPIRATION,
+            )
+            .unwrap();
+        claims
+            .create_claim(
+                deps.as_mut().storage,
+                &Addr::unchecked("addr2"),
+                (TEST_AMOUNT + 100).into(),
+                TEST_EXPIRATION,
+            )
+            .unwrap();
+
+        let cls = claims
+            .range_claims(deps.as_ref(), None, None, Order::Ascending, None)
+            .unwrap();
+        let expected = vec![
+            (
+                Addr::unchecked("addr"),
+                vec![Claim {
+                    amount: TEST_AMOUNT.into(),
+                    release_at: TEST_EXPIRATION,
+                }],
+            ),
+            (
+                Addr::unchecked("addr2"),
+                vec![Claim {
+                    amount: (TEST_AMOUNT + 100).into(),
+                    release_at: TEST_EXPIRATION,
+                }],
+            ),
+        ];
+        assert_eq!(cls, expected)
+
+        // no need test intensively, storage range works
```

Comment:
This is range query where you don't test range at all.
There are 3 Option parameters, this is 1 of potentially 9 possible cases (or more, if we'd like to cover order as well).
Why don't you create additional test cases at least for pagination?

I get that probably similar implementation is tested elsewhere, but this is open sourced - IMO we should cover more.